onnxruntime/core/providers/nnapi/nnapi_builtin/builders/op_support_checker.h

```diff
@@ -9,13 +9,15 @@ namespace onnxruntime {
 namespace nnapi {
 
 struct OpSupportCheckParams {
-  OpSupportCheckParams(int32_t android_feature_level, bool use_nchw)
+  OpSupportCheckParams(int32_t android_feature_level, bool use_nchw, std::vector<const Node*> dq_nodes_in_group)
       : android_feature_level(android_feature_level),
-        use_nchw(use_nchw) {
+        use_nchw(use_nchw),
+        dq_nodes_in_group(dq_nodes_in_group) {
```

Comment:
can do `std::move(dq_nodes_in_group)` to move instead of copy, or pass parameter dq_nodes_in_group by const reference